impl/build.gradle

```diff
@@ -32,11 +32,14 @@ dependencies {
   implementation (group: 'org.ehcache', name: 'sizeof', version: parent.sizeofVersion) {
     exclude group:'org.slf4j', module:'slf4j-api'
   }
-  implementation ("org.terracotta:statistics:$parent.statisticVersion")
+  implementation ("org.terracotta.management:management-model:$terracottaPlatformVersion") {
```

Comment:
This bothers me a little too (although less than the core example). Also - why are we excluding the transitive statistics dependency?

Reply:
Because we only want ehcache modules to use stats api exposed by core module, rather than directly using them, removing transitive dependency makes sure that no stats api was directly being used from stats lib